beacon-chain/sync/subscriber.go

```diff
@@ -3,6 +3,7 @@ package sync
 import (
 	"context"
 	"fmt"
+	"github.com/prysmaticlabs/prysm/v3/config/features"
 	"reflect"
 	"runtime/debug"
 	"strings"
```

Comment:
Please put this import with the other non standard library imports, below line 12

Reply:
fixed